app/models/run.rb

```diff
@@ -93,6 +93,7 @@ def parse(fast: true, convert: false)
         name: result[:name],
         program: result[:program],
         attempts: result[:attempts],
+        srdc_id: result[:srdc_id].blank? ? nil : result[:srdc_id],
```

Comment:
This can be simplified with `presence`: http://api.rubyonrails.org/v4.2.5/classes/Object.html#method-i-presence